core/v2/types.go

```diff
@@ -72,6 +72,9 @@ type BlobHeader struct {
 
 	// Signature is the signature of the blob header by the account ID
 	Signature []byte
+
+	// Salt is used to make blob intentionally unique when everything else is the same
+	Salt uint32
```

Comment:
move salt above signature. also make signature's comment more precise. What EXACLY is it signing over?